src/Assets/pageTitleTags.ts

```diff
@@ -0,0 +1,17 @@
+const pageTitleTags: Record<string, string> = {
```

Comment:
If you want to follow the constant naming convention, `pageTitleTags` is also a constant.